### DISCUSSION TITLE
Subtle background gradient rays

### DISCUSSION THREAD

**Discussion by mvpcurtis** [tailwindlabs/tailwindcss#12987] [Help] [answered]
Hi all.

I've recenrty started using TailwindCSS I think I'm falling in love with it. The reason why I'm here today is to ask if anyone knows how to recreate that subtle gradient effect they have in their docs page. I've looked through the code and I can't seem to locate anything regarding to it.

Please be patient with me and thank you :)

<img width="955" alt="Screenshot 2024-02-20 at 20 54 10" src="https://github.com/tailwindlabs/tailwindcss/assets/16779611/1b1e2215-fc83-42b0-9aec-22186e9d6bc1">

**Comment by wongjn** [comment, marked as answer]
It's these elements:
![image](https://github.com/tailwindlabs/tailwindcss/assets/11310624/77e9a441-506e-4e2c-b277-2aa4b13da0ad)

Here that is, [recreated in a Tailwind Play](https://play.tailwindcss.com/NvsEdacAXu).

**Comment by mvpcurtis** [comment]
Thank you for the reply. I've been browsing the stylesheets for a while looking for gradients. I didn't realise they use images. Again, thank you.
